mmv1/api/resource.go

```diff
@@ -1293,6 +1293,17 @@ func (r Resource) GetPropertyUpdateMasksGroups() map[string][]string {
 	return maskGroups
 }
 
+func FormatDocDescription(desc string) string {
```

Comment:
Will it remove all of diffs in the description field or it works for some use cases? 
A comment could be helpful.

Reply:
It should be all diffs in the documentation description, the Ruby template has specific indentation that this copies.
Added a comment